client2/src/components/game/gui/GUI.scss

```diff
@@ -86,3 +86,23 @@
   transform: translateX(50%);
   margin-bottom: 30px;
 }
+
+#chat-input {
```

Comment:
@Arcanorum did I place the this css in the correct file?

Reply:
I think you should make a separate component for the chat box stuff and keep the style with it.

Reply:
WIll Do 💯